### ISSUE TITLE
typos

### ISSUE THREAD

**Issue by cmmizer** [MicrosoftDocs/sql-docs#2245]
"retry_attemptsis int" =&gt; "retry_attempts is type int"
"retry_intervalis int" =&gt; "retry_interval is type int"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 571ba504-da27-884c-e533-e4671b00ad66
* Version Independent ID: 87f89fc3-ba62-66ff-5651-3a8091ee6a3e
* Content: [sp_add_jobstep (Transact-SQL) - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/sp-add-jobstep-transact-sql?view=sql-server-2017#feedback)
* Content Source: [docs/relational-databases/system-stored-procedures/sp-add-jobstep-transact-sql.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/system-stored-procedures/sp-add-jobstep-transact-sql.md)
* Product: **sql**
* Technology: **system-objects**
* GitHub Login: @stevestein
* Microsoft Alias: **sstein**

**Comment by MightyPen** [comment]
Fixing with PR 10511:
https://github.com/MicrosoftDocs/sql-docs-pr/pull/10511

**Comment by MightyPen** [comment]
Fixed.

@cmmizer Hi, Thank you for bringing this issue to our attention. Obviously tipped by this Issue 2245, I found several more occurrences of the same two problem occurrences that you reported in the original description, and I fixed them too.

Customers such as you have the authority to edit our documentation, by clicking the 'Edit-pencil' icon at the top-right on the document's webpage. We encourage customers to use the Edit approach (as an alternative to this Issue approach) whenever they know the needed textual fix. Feel free to use the Edit option in cases where you are comfortable with it.

Thanks again for creating this Issue 2245.
I am now Closing this Issue 2245.
